packages/client/components/AnalyticsPage.tsx

```diff
@@ -57,17 +59,21 @@ if (__PRODUCTION__ && datadogClientToken && datadogApplicationId && datadogServi
   })
   datadogRum.startSessionReplayRecording()
 }
+
 // page titles are changed in child components via useDocumentTitle, which fires after this
 // we must guarantee that this runs after useDocumentTitle
 // we can't move this into useDocumentTitle since the pathname may change without chaging the title
 const TIME_TO_RENDER_TREE = 100
 
 const AnalyticsPage = () => {
-  const key = window.__ACTION__.segment
-  if (!key) return null // development use
```

Comment:
Since this "Page" is doing more than just dealing with Segment, I'm disabling on the `__PRODUCTION__` flag instead of the presence of the key.